pkg/policyreport/reportrequest.go

```diff
@@ -28,6 +29,8 @@ const (
 	workQueueRetryLimit = 10
 )
 
+type concurrentMap struct{ cmap.ConcurrentMap }
```

Comment:
why are we redeclaring this type?

Reply:
We define custom methods on this:
https://github.com/kyverno/kyverno/pull/4148/files/3d500c9be1538f9ae3a633eb336bfbca6bfff86e#diff-a6a38567be30c7fe9570b86c1223556678ce0cdb38f1425c4e2e28ec3a1afbf2R348-R357

Reply:
Ok. Maybe good to move the type and methods to a separate file, so its clear.

Reply:
Done! Needs re-approval :)

Reply:
Did you add the new file?

Reply:
Ops.. missed it. Added now!